README.md

```diff
@@ -1,3 +1,55 @@
+# icb-website-react
+
+## Setup
+
+- Install npm
+
+- TODO
+
+## Workflow
+
+**You MUST be familiar with pull requests and git basics before you can work on this project.**
+
+> We will use git as our version-control system. If you don't know what git is, read more [here](https://www.freecodecamp.org/news/learn-the-basics-of-git-in-under-10-minutes-da548267cc91/).
+
+> We will be using pull-requests to manage this project. If you do not know what a pull-request is, read more [here](https://learn.co/lessons/github-pull-request-basics).
+
+If you have any questions about this process, please use the internet as a resource. If that doesn't work please ask for help from someone who knows how this stuff works.
+
+### Getting started
+
+- If you have never worked on the project before, start by cloning the repository onto your local machine.
+  - In your terminal on your computer, navigate to the directory (folder) where you want the project to live
+  - In this directory, run the command `git clone https://github.com/inventioncorps/icb-website-react.git icb-website`
```

Comment:
Requires some more detailed instruction. Perhaps include:
1. git init (Initialize local repository)
2. git clone ... (clone remote repository into folder named icb-website)
3. cd icb-website
4. git branch [personal-branch-name] (create local branch)
5. git checkout [personal-branch-name] (go to new branch)
6. git push -u origin [personal-branch-name] (create branch in remote repository; This step may require auth token)

Reply:
I don't think git init is needed if in this case?

If I am in the MyProjects directory and I run `git clone ...` it creates a git repository for me.

I will add something about entering the new repository.